install-macos.sh

```diff
@@ -26,10 +26,13 @@ You must specify the VANTA_KEY environment variable in order to install the agen
     exit 1
 fi
 
-if [ $(/usr/bin/arch) == "arm64" ] && [ ! -f "/Library/Apple/System/Library/LaunchDaemons/com.apple.oahd.plist" ]; then
+
+if [ $(/usr/bin/arch) == "arm64" ] && ! /usr/bin/pgrep oahd >/dev/null 2>&1; then
     printf "\033[31m
 You must set up Rosetta on your Mac in order to install the agent. Please
 follow the setup instructions from Apple here: https://support.apple.com/en-us/HT211861.
```

Comment:
This sentence should probably now be "You can find information about Rosetta here:".